src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowsServices/Properties/Resources.Designer.cs

```diff
@@ -0,0 +1,333 @@
+﻿//------------------------------------------------------------------------------
```

Comment:
@crutkas I need a C# adult - this looks like the kind of file that should be .gitignored?